src/metadata/metacontent_handler.cc

```diff
@@ -187,11 +195,17 @@ void ContainerArtHandler::fillMetadata(std::shared_ptr<CdsObject> item)
     // auto tweak = config->getDirectoryTweakOption(CFG_IMPORT_DIRECTORIES_LIST)->get(item->getLocation());
     // auto path = getContentPath(tweak == nullptr || !tweak->hasContainerArtFile() ? names : std::vector<std::string> { tweak->getContainerArtFile() }, item, tweak != nullptr && tweak->hasCaseSensitive() ? tweak->getCaseSensitive() : caseSensitive);
     auto path = getContentPath(names, item, caseSensitive, config->getOption(CFG_IMPORT_RESOURCES_CONTAINERART_LOCATION));
-    log_debug("Running ContainerArt handler on {}", !path.empty() ? path.c_str() : item->getLocation().c_str());
+    log_info("Running ContainerArt handler on {}", !path.empty() ? path.c_str() : item->getLocation().c_str());
 
     if (!path.empty()) {
         auto resource = std::make_shared<CdsResource>(CH_CONTAINERART);
-        resource->addAttribute(R_PROTOCOLINFO, renderProtocolInfo("jpg"));
+        std::string type = path.extension().string().substr(1);
+#ifdef HAVE_MAGIC
+        std::string mimeType = mime->fileToMimeType(path);
+#else
+        std::string mimeType = fmt::format("image/{}", type);
```

Comment:
I guess we should should run this through the mapping configuration too?

Possibly worth pulling it out into a separate method for use here and in the main import flow?

Reply:
It got a taste of spring cleaning 😄